java-plugin/src/test/java/fr/greencodeinitiative/java/checks/CookieWithExpirationCheckTest.java

```diff
@@ -0,0 +1,20 @@
+package fr.greencodeinitiative.java.checks;
```

Comment:
please add a new test file in `ecoCode-java-test-project` repository and check all is ok with an local instance